scripts/tests/cirque_tests.sh

```diff
@@ -16,17 +16,26 @@
 # limitations under the License.
 #
 
-set -x
-
 SOURCE="${BASH_SOURCE[0]}"
 SOURCE_DIR="$(cd -P "$(dirname "$SOURCE")" >/dev/null 2>&1 && pwd)"
 REPO_DIR="$SOURCE_DIR/../../"
+TEST_DIR=$(readlink -f "$REPO_DIR"/src/test_driver/linux-cirque)
```

Comment:
The `-f` argument to `readlink` is not portable, as far as I can tell.  It exists on Linux, but not on BSD (and hence not on MacOS).

Is this script expected to be run outside one of our Linux devcontainers?

If we already require Python anyway, we could use something like:
```
TEST_DIR=$(python -c "import os,sys; print(os.path.realpath(os.path.expanduser(sys.argv[1])))" "$REPO_DIR"/src/test_driver/linux-cirque)
```
which might be more portable.

That said, it's not obvious to me why we need to do this manual symlink expansion at all.  The only use of `TEST_DIR` I see is as an argument to `cd`, which would handle symlinks anyway...

Reply:
cirque can only be setup in Linux, cirque's virtual wifi, thread, later ble are relying on linux modules, mac80211_hwsim, otbrposix, btvirt along with bluetoothd.

Cirque is the framework orchestrating virtual docker based devices interaction, several virtualization modules needs to be configured in host, instead of our Linux devcontainers? Yes, currently it is expected to be run outside one of our linux devcontainers.